src/app/models/book-toc.ts

```diff
@@ -3,13 +3,10 @@ import {cnxFetch} from './table-of-contents-html';
 import memoize from 'lodash/memoize';
 
 export function bookToc(slug: string) {
-    if (!slug) {
```

Comment:
Is this guaranteed to not be empty, or maybe prefer to have the throw in `cmsFetch`?

Reply:
The [only place it is called](https://github.com/openstax/os-webview/blob/main/src/app/pages/errata-form/form/ErrorLocationSelector/toc-selector.js#L93) tests for `slug`.